participants/Zangshell virtual reality.md

```diff
@@ -0,0 +1,102 @@
+#  Zangshell virtual reality and Augmented reality system 
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Please tell us your project's name and the Filecoin Slack handles for your teammates.*
+
+Project Name: Zangshell virtual reality and Augmented reality system
+
+Slack user: Yulin578873638
+
+## (1) Brief Project Description
+
+*Add your project's description here.* 
+
+Zangshell is a kind of virtual reality(VR) and Augmented reality(AR) system, in the past,  it mainly served the real estate industry and municipal engineering. the orginal data was stored in an Internet Data Ceter(IDC) before. Due to higher costs, the company ower is intended to migrate the data to IPFS. 
+
```

Comment:
Please note that the data should be on the Filecoin network for competition eligibility.